cSpell.json

```diff
@@ -0,0 +1,84 @@
+{
+	"version": "0.2",
+	"language": "en",
+	"words": [
+		"Codegen",
+		"geolocation",
+		"iframes",
+		"navigations",
+		"pluggable",
+		"Polyfilling",
+		"Pytest",
+		"sandboxing",
+		"SIGHUP",
+		"SIGTERM",
+		"stringifiable"
+	],
+	"ignoreWords": [
```

Comment:
ℹ️ Note; `words` can be suggested when spellcheck detects a misspelling in the IDE. `ignoreWords` won't be suggested.